examples/asr/librispeech_biasing/global_stats_100.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
Is it computed based on only "train-clean-100" subset? The file name need to match the default one in `train.py`

Reply:
It is. Because I was only able to perform experiments on train-clean-100 at the moment, but with the help from others it should finally be on 960-hour. So should I change the default in train.py to train-clean-100 here for this PR?

Reply:
I see. In that case we can stick with `global_stats.json` for both file name and the default value in the train.py for simplicity. Just make sure the values are updated here when you update the WER scores for 960 hours of training data.